core-v1/src/main/scala/dsl/JustifiedBuildExprDsl.scala

```diff
@@ -3,13 +3,20 @@ package com.rallyhealth.vapors.v1
 package dsl
 
 import algebra._
-import data.{FactTypeSet, Justified}
-import logic.Logic
-
 import cats.data.NonEmptyList
 import cats.{Foldable, Functor}
+import data.{FactTypeSet, Justified, Window}
+import lens.VariantLens
+import logic.Logic
+
+trait JustifiedBuildExprDsl extends BuildExprDsl with WrapJustifiedImplicits with JustifiedDslTypes {
 
-trait JustifiedBuildExprDsl extends WrappedBuildExprDsl with JustifiedDslTypes {
+  implicit def wrapWindow[O](window: Window[O])(implicit opW: OP[Justified[Window[O]]]): Any ~:> Justified[Window[O]] =
+    Expr.Const(wrapConst.wrapConst(window))
+
+  // TODO: Remove the need for the above line ^
```

Comment:
I should either move this to the `BuildExprDsl` or just require using `Window.between(x, y).const`... or maybe some other syntax, but still using `.const`